pkg/networkmanager/dialogs-common.jsx

```diff
@@ -248,9 +248,9 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
             dlg = <IpSettingsDialog topic="ipv6" {...properties} />;
 
         if (dlg)
-            resolveDeps(type).then(() => {
-                Dialogs.show(dlg);
-            }).catch(console.error); // not-covered: OS error
+            resolveDeps(type)
+                    .then(() => Dialogs.show(dlg))
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)